docs/Documentation.md

```diff
@@ -273,7 +291,8 @@ A default configuration file would look like
   "SpaceAroundDelimiter":true ,
   "KeepNewlineAfter":false,
   "MaxIfThenElseShortWidth":40,
-  "StrictMode":false
+  "StrictMode":false,
```

Comment:
Nitpick, keep StrictMode as the last setting, add `EmptyLineBeforeNestedModuleBody` above it.
StrictMode is bit of a special setting, it shouldn't be used by end-users.